### ISSUE TITLE
How do I find out the embedded Antlr version?

### ISSUE THREAD

**Issue by serefarikan** [mike-lischke/vscode-antlr4#311]
Hi, I read in a tutorial (sorry, lost the link) that this extension contains an embedded version of antlr and that my project should rather use the same version with the embedded one. 
I'd like to follow that advice but I could not figure out how to find the embedded antlr version. Any help would be much appreciated.

**Comment by mike-lischke** [comment]
No, this extension does not include an own ANTLR4 jar. Instead it uses [antlr4ng](https://github.com/mike-lischke/antlr4ng), which includes a custom build of ANTLR 4.13.2.
